.gitignore

```diff
@@ -0,0 +1,2 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
FYI, it is a good practice to split list of changes using commits. This simplifies review and keep attention on important topics.
E.g.
1 commit: update quotes
2 commit: add support of fixtures
3 commit: something else

At the review you will see squashed changes, but if you want review particular topic you can do this with approach above, with big commit like yours it is not possible.

Reply:
I completely agree. Thanks for the advice.

Reply:
I will follow this advice in next time.